GitExtUtils/GitUI/ThreadHelper.cs

```diff
@@ -58,7 +59,7 @@ public static void ThrowIfNotOnUIThread([CallerMemberName] string callerMemberNa
                 return;
             }
 
-            if (!JoinableTaskContext.IsOnMainThread)
+            if (!JoinableTaskContext!.IsOnMainThread)
```

Comment:
💡 All of the `!` lines for this property can be removed if the property is changed to not return null.